src/plugins/data/common/query/filters/persistable_state.ts

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
This file is moved as is

`src/plugins/data/common/query/persistable_state.ts` -> `src/plugins/data/common/query/filters/persistable_state.ts`